src/models/refreshToken.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Add this rule in the `.eslintrc.json` file, instead of disabling it here.

Reply:
@samuel-shyaka-dus Thanks for the review but if that rule was to be added in the whole project, we may push code with some modules not installed. As in this file the rule is not raising any error probably was added by mistake or before a module was installed. The line should be removed as a whole

Reply:
Okay